### PR TITLE
add the usage information for for git-branch --set-upstream-to

### LINKED ISSUE #130: Add the use cases for git-branch --set-upstream-to
I just needed to find out how to set the upstream to a local branch and I couldn't find it here. It should probably be added here.

### PR DESCRIPTION
**Pull request by icyflame**
Fix #130

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -789,6 +790,23 @@ $ git push
...
 The behavior of the other modes of ```git push``` is described in the doc of push.default.
 
+<a name="i-want-to-set-a-remote-branch-as-the-upstream-for-a-local-branch"></a>
+### I want to set a remote branch as the upstream for a local branch
+
+You can set a remote branch as the upstream for the current local branch using:
+
+```sh
+$ git branch --set-upstream-to [remotename]/[branch]
+$ or, using the shorthand:
```
Comment (RichardLitt) [inline README.md]: This line should be in a `#` or outside of the codeblock.

Comment (icyflame) [reply README.md]: OH! I am extremely sorry, this should be in a bash comment indeed. I have fixed it now.

**Comment by RichardLitt** [comment]
Great work! Thanks!

**Comment by icyflame** [comment]
Thanks a lot for merging this in, @RichardLitt! :+1: :smile:

**Comment by RichardLitt** [comment]
Thank you for making it!

**Pull request merged**